lib/workflow/factory.js

```diff
@@ -5,15 +5,15 @@ var Node = require('../node/node');
 var async = require('async');
 var debug = require('../util/debug')('factory');
 
-var typeNodeMap = nodes.reduce(function(typeNodeMap, node) {
+var DEFAULT_TYPE_NODE_MAP = nodes.reduce(function(typeNodeMap, node) {
```

Comment:
Two things:
 - We want to have them indexed by type.
 - The `../node/nodes` directory doesn't have an `index.js`, so it won't provide us with the nodes list.